homework/index.js

```diff
@@ -123,79 +125,75 @@
   // Creates html elements from an object
   function createStaticComponentsFromModel(model, root) {
```

Comment:
https://github.com/HackYourFuture/JavaScript3/pull/253#discussion_r294050179

I tried to make challenge in line with your recommendations in the link but I noticed that my approach didn't make sense. It should act as virtual dom. Otherwise, whenever I change the select element, it would re-create all the parts. 

I should focus the goal of the homeworks. I accept that my approach was more than enogh. Therefore, I left the model part and changed the creating elements part into traditional approach.

Reply:
I'm happy to accept your solution as-is.